src/us/kbase/workspace/test/database/provenance/CreditMetadataTest.java

```diff
@@ -263,6 +280,8 @@ private void assertCreditMetadataFields(final CreditMetadata cm,
 			final List<PermanentID> relatedIdentifiers) {
 
 		assertThat(INCORRECT_ID, cm.getIdentifier(), is(identifier));
+		assertThat(INCORRECT_REPO, cm.getRepository(), is(repository));
+		assertThat(INCORRECT_ID, cm.getDescription(), is(description));
```

Comment:
This looks incorrect - there's another description checking line below. C&P issue?

Reply:
Indeed it is. Fixed.

Reply:
:+1: